### DISCUSSION TITLE
Hero11 at 5.3k & 120fps? (Fishing)

### DISCUSSION THREAD

**Discussion by Ferio12** [gopro/labs#425] [Q&A]
Hi Everyone,

I use my gopro for fishing and like to shoot in a higher FPS to get a better slowmo of surface attacks, yet in 5k not to lose that much quality with a zoom in. I use goprolabs now to have hindsight always on, not to miss any bites as it used to turn off after 15 minutes. I pretty much want to have just 3 presets

1) 5k/120 fps/hindsight always on (with adding slowmo & zoom in davinci)
2) 5k/60fps (just for shooting general/zoom)
3) Photo's

With the QR generator I can record at 5k/120fps with goprolabs installed, is this possible? Will it work? Do I need to set other settings that I'm not aware of? Better stick with 4k/120fps or 5k/60fps? What about bitrate settings?

I'm new to this, so learning. ;) 

Here's my QR
![Capture](https://github.com/gopro/labs/assets/137981912/beee4ef1-8a88-4366-944c-f72b4e157576)

**Comment by dnewman-gpsw** [comment]
The camera doesn't have 5k/120fps, even though you can create the QR code.   4K/120 for the best slow motion or 5K/60 for a tad more punch in and good slow motion.

Bitrate extensions beyond 120Mb/s are not active in Hindsight.
